styles.css

```diff
@@ -0,0 +1,18 @@
+body {
+    background-color: white;
+}
+
+.ejemplo{
+    display: flex;
+    justify-content: center;
+	align-items:center;
```

Comment:
the indentation of this 2 properties is not the same as the rest. also leave a space after the :